.github/workflows/provider-compatibility.yml

```diff
@@ -10,9 +10,7 @@
 
 name: Provider compatibility across versions
 
-on: #[pull_request]
-  schedule:
-    - cron: '0 15 * * *'
+on: [pull_request, push]
```

Comment:
I think running it on pull request is too much.

Reply:
There is no other way to test it.

I think it should remain daily.

Reply:
Yeah, then please add fixup commit reverting this back to scheduled